### ISSUE TITLE
HTTP Error 404 - After a successful "New Azure OpenAI" deployment to Azure using "Deploy to Azure" button

### ISSUE THREAD

**Issue by bsonnek** [microsoft/semantic-kernel#1203]
I ran the "Deploying with a new Azure OpenAI instance"  Azure Portal command from here:
https://github.com/microsoft/semantic-kernel/blob/main/samples/apps/copilot-chat-app/webapi/DeploymentTemplates/README.md

The deployment was successful in the Azure Resource Group deployment templates. 

When I test the web app with  the   https://app-skweb.azurewebsites.net/probe
I receive "Semantic Kernel Service up and running"

But when I try to access the chat web app for the first time I receive an HTTP Error 404 - page can’t be found.
Accessing this link gets 404 - https://app-skweb.azurewebsites.net

I'm running this directly from the main github repo so everything is recent.

Any Idea what I'm missing? 

Thanks in advance!

**Comment by adrianwyatt** [comment]
The templates only deploy backend web api (i.e., CopilotChatWebApi). For the end-to-end experience you'll need to run the frontend web app (i.e., `samples\apps\copilot-chat-app\webapp`) and configure it to use your instance of the backend webapi. I think the issue here is that we're missing a section in the READMEs on connecting these two. 

1. Get your API key from your deployed instance (https://github.com/microsoft/semantic-kernel/blob/main/samples/apps/copilot-chat-app/webapi/DeploymentTemplates/README.md#authorization)
2. Follow the instructions to run the frontend web app locally and configure it to use your deployed backend: https://github.com/microsoft/semantic-kernel/blob/main/samples/apps/copilot-chat-app/README.md#start-the-webapp-frontend-application. 

We'll take a look at making this process more obvious this week.

**Comment by bsonnek** [comment]
Great! Thank you for the quick response!

**Comment by bsonnek** [comment]
One last question - for modifying the .env file with the backend URI what should it be? The Azure WebAPI server doesn't appear to allow traffic on port 40443 - I've opened ports on the NSG and still cannot get a response. 

I've tried different variations of the URI but can't get my webapp to open up. It just sits and spins..

URIs I've tried:
http://app-semkernel-skweb.azurewebsites.net:40443/
https://app-semkernel-skweb.azurewebsites.net:40443/
https://app-semkernel-skweb.azurewebsites.net
